### ISSUE TITLE
Feature request: time scrubbing in the editor

### ISSUE THREAD

**Issue by hybridherbst** [google/model-viewer#3432]
### Description

When testing longer animations, it would be nice to be able to scrub time in modelviewer.dev/editor. This exists (and is very useful) on https://sandbox.babylonjs.com/ but since there's differences in behaviour that's not always enough to test if a file works correctly in model-viewer.

Related https://github.com/mrdoob/three.js/issues/24018

**Comment by elalish** [comment]
We have an animation scrubber, you just have to pause it (uncheck play). Is anything else missing?

**Comment by hybridherbst** [comment]
Woah 🤯 I'd have never found that by myself, that is _very_ hidden (and a tiny slider)!

This definitely helps, but I wonder why it's not in a more prominent place when a file is animated (e.g. over the entire width of the bottom), and why it can't be scrubbed while playing? While the slider is dragged the animation would need to pause of course.

For context: when I drop a file with animation that just starts playing - I did that hundreds of times already but it never crossed my mind to switch tabs, _pause_ this, and then be able to scrub. Also, animations can be minutes (see the linked file in the issue above), so the tiny slider is really hard to use (but kind of works!).

**Comment by elalish** [comment]
Yeah, you might notice the editor is not exactly our *primary* product... :) The fact we have it at all was a bit of a coup. It's very much a spare time project for me and there's zero UX guidance, so what you see is generally how I can get a feature in with absolute minimum effort. Anyway, contributions welcome!
